pyproject.toml

```diff
@@ -71,10 +61,23 @@ update-contributors = "pyosmeta.cli.update_contributors:main"
 update-reviews = "pyosmeta.cli.process_reviews:main"
 update-review-teams = "pyosmeta.cli.update_review_teams:main"
 
+### Hatch config ###
+
 [tool.hatch]
 version.source = "vcs"
 build.hooks.vcs.version-file = "src/pyosmeta/_version.py"
 
+[tool.hatch.envs.test]
+dependencies = [
+  "pytest",
+  "pytest-cov",
+]
+
+[tool.hatch.envs.test.scripts]
```

Comment:
this sets up a script that like nox can be run. 

so running `hatch run test:run-tests`
where test is the envt name and run-tests is the script to call will run our tests and make sure pytest is installed. you can also setup envt defaults if you want